src/usdb_syncer/resource_dl.py

```diff
@@ -79,12 +79,23 @@ def download_video(
     if browser.value:
         ydl_opts["cookiesfrombrowser"] = (browser.value,)
     if isinstance(options, AudioOptions):
-        postprocessor = {
+        pp_extract_audio = {
             "key": "FFmpegExtractAudio",
             "preferredquality": "320",
             "preferredcodec": options.format.value,
         }
-        ydl_opts["postprocessors"] = [postprocessor]
+        pp_normalize = {
+            "key": "ExecAfterDownload",
+            "exec_cmd": " && ".join(
+                [
+                    # simple loudness normalization with EBU R128 algorithm
+                    # see https://trac.ffmpeg.org/wiki/AudioVolume
+                    f"ffmpeg -i {path_stem}.%(ext)s -filter:a loudnorm temp-audio.%(ext)s",
+                    f"mv test-audio.tmp.%(ext)s {path_stem}.%(ext)s",
```

Comment:
Haven't had a chance to try this out, yet, but doesn't `path_stem` have to be escaped? And I think the temp file should be created in the same folder, or else there may be issues when moving across filesystems or crashing bundles as in #37.

Reply:
Additionally, we should be careful to use OS-specific commandline arguments as little as possible. I would suggest at this point a python [tempfile ](https://docs.python.org/3/library/tempfile.html)and the [shutil ](https://docs.python.org/3/library/shutil.html)library for file handling :)